[docs] Revise and split up "Overview" page into "Introduction" section (#4692)
mui/mui-x: docs/data/introduction/support/support.md

# Support

> <p class="description">Learn how to get support for MUI X components, including feature requests, bug fixes, answers to how-to questions, and professional support from our team.</p>

Reviewer: ```suggestion
<p class="description">How to get support for MUI X components, including feature requests, bug fixes, answers to how-to questions, and professional support from the team.</p>
```

## GitHub

We use GitHub issues as a bug and feature request tracker.

> If you think you've found a bug, or you have a new feature idea, please start by making sure it hasn't already been [reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed).

Reviewer: ```suggestion
If you think you've found a bug, or you have a new feature idea, please start by [making sure it hasn't already been reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed).
```

You can search through existing issues and pull requests to see if someone has reported one similar to yours.

[Open an issue](https://github.com/mui/mui-x/issues/new/choose) in the MUI X repo.

## Stack Overflow

Visit Stack Overflow to ask questions and read crowdsourced answers from expert developers in the MUI community, as well as MUI maintainers.

[Post a question about MUI X](https://stackoverflow.com/questions/tagged/mui) on Stack Overflow.

## Professional support

> When purchasing an MUI X Pro or Premium license you get access to professional support for a limited duration. Support is available on multiple channels, but the recommended channels are:

Reviewer: ```suggestion
When purchasing a MUI X Pro or Premium license you get access to professional support for a limited time. Support is available on multiple channels, but the recommended channels are:
```

- GitHub: You can [open a new issue](https://github.com/mui/mui-x/issues/new/choose) and leave your Order ID, so we can prioritize accordingly.

> - Email (only if your issue requires sharing private information): You can [submit a request](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email to [x@mui.com](mailto:x@mui.com).

Reviewer: ```suggestion
- Email: If you need to share private information you can [submit a request](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email to [x@mui.com](mailto:x@mui.com).
```

> Your Order ID on the issue helps us prioritize the issues based on the following support levels:

Reviewer: ```suggestion
Including your Order ID in the issue helps us prioritize the issues based on the following support levels:
```

> 1. MUI X Pro: MUI's maintainers give these issues more attention than the ones from the Community plan.

Reviewer: ```suggestion
1. MUI X Pro: maintainers give these issues more attention than the ones from the community.
```

> 2. MUI X Premium: Same as MUI X Pro, but with priority over Pro.

Reviewer: ```suggestion
2. MUI X Premium: The same as MUI X Pro, but with higher priority.
```

> 3. MUI X Priority support add-on (not available yet): A provided SLA with 24h for the first answer.

Reviewer: ```suggestion
3. MUI X Priority support add-on (coming soon): Provides a 24h SLA for the first answer.
```
